cassandra/cluster.py

```diff
@@ -765,6 +765,10 @@ def set_min_requests_per_connection(self, host_distance, min_requests):
             raise UnsupportedOperation(
                 "Cluster.set_min_requests_per_connection() only has an effect "
                 "when using protocol_version 1 or 2.")
+        if min_requests < 0 or min_requests > 126 or \
+           min_requests >= self._max_requests_per_connection[host_distance]:
+            raise ValueError("min_requests must be in the range [0, 127) and less than the max_requests for this host_distance (%d)" %
```

Comment:
That's interval notation: https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints
If you think it's too confusing I could change it. Realistically these parameters should never be near the endpoints though.